app/main.py

```diff
@@ -1,6 +1,22 @@
+import requests
+import os
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(F"{URL}?key={API_KEY}&q={CITY}&aqi=no").json()
```

Comment:
try to use `params=` in requests.get() instead of write them in ulr with f string